README.md

```diff
@@ -44,6 +44,7 @@ The following modules are supported by PyQt5-stubs:
 # Building upstream stubs
 The Dockerfile is used to build all of the stubs for the upstream PyQt5 modules.
 The Dockerfile consists of multiple build layers:
+layers:
```

Comment:
I think this is an unintentional merge artifact?

Reply:
```suggestion
```
Quite so.